packages/quicko-webapp/src/shared/utils.ts

```diff
@@ -12,11 +12,11 @@ export const convertStoreToFileText = (store: RootState) => {
     const text = Object.keys(store).reduce(
```

Comment:
The variable is named text but it's not really a text :/ What about stringifiedStoreItems ?